README.md

```diff
@@ -1 +1,47 @@
-# Es6_AwesomeBooks
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Es6_AwesomeBooks
+> A book collection store using Es6 and arrow functions
+
+## Built With
+- HTML
+- CSS
+- JS
+
+<!-- ## Live Demo -->
+
+### Prerequisites
+The basic requirements for building the executable are:
+* A working browser application (Google chrome, Mozilla Fire fox, Microsoft edge ...)
+
+## Getting Started
+
+#### Cloning the project
+```
+https://github.com/Yorsyboy/Es6_AwesomeBooks.git
+```
+## Authors
+
+## Author 1
```

Comment:
- [ ] please make sure to use [the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.